docs/tex/tutorials/linear-mixed-effects-models.tex

```diff
@@ -0,0 +1,104 @@
+\title{Linear Mixed Effects Models}
+
+\subsection{Linear Mixed Effects Models}
+
+In linear mixed effects models, we wish to model a linear relation between categorical input variables and a continuous output variable. The data is not drawn independently. Oftentimes, we see clusters in subgroups.
```

Comment:
are the inputs necessarily categorical? I think you mean something like "[...] we wish to model a linear relationship for data points with inputs of varying type, categorized into subgroups, and associated to a continuous output".